spec/formatting.md

```diff
@@ -2,13 +2,15 @@
 
 ## Introduction
 
-This document defines the behaviour of a MessageFormat 2.0 implementation
+This document defines the behavior of a MessageFormat 2.0 implementation
 when formatting a message for display in a user interface, or for some later processing.
 
-To start, we presume that a _message_ has either been parsed from its syntax
-or created from a data model description.
-If this construction has encountered any _Syntax Errors_ or _Data Model Errors_,
-an appropriate error MUST be emitted and a _fallback value_ MAY be used as the formatting result.
+To start, parse the _message_ from its syntax or create it from a data model description.
+
+If the message being formatted has any _Syntax Errors_ or _Data Model Errors_,
```

Comment:
Should keep the bit about emitting the error. I'm also not really sure about the "message has errors" language here; I think the previous language on this was less opinionated about error handling.

```suggestion
If the message being formatted has any _Syntax Errors_ or _Data Model Errors_,
an appropriate error MUST be emitted and
```

Reply:
I'm not sure what else we would really say here? 

We could also do:

```suggestion
If the _message_ being formatted contains a _Syntax Error_ or _Data Model Error_
emit an appropriate error.
The result of formatting a _message_ containing such an error is a single _fallback value_
that uses the _message_'s fallback string defined in the _formatting context_
or, if this is not available or empty, the U+FFFD REPLACEMENT CHARACTER `�`.
```